src/test/resources/assets/trapdoor_test/blockstates/trapdoor_test_ladder.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
DataGens are preferred , even for test mods.

Reply:
applied.